templates/react-native-template-realm-ts/template/app/models/Task.ts

```diff
@@ -1,32 +1,51 @@
-import { Realm, createRealmContext } from "@realm/react";
+////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2021 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////
+import {Realm, createRealmContext} from '@realm/react';
 export class Task extends Realm.Object {
   _id!: Realm.BSON.ObjectId;
   description!: string;
   isComplete!: boolean;
+  userId!: string;
 
-  static generate(description: string) {
+  static generate(userId: string | undefined, description: string) {
```

Comment:
Optional arguments should come last.  Otherwise when userId is not defined it will need to be called like this:
```
generate(undefined, "task description")
```